src/parallel/HighsTaskExecutor.h

```diff
@@ -129,7 +129,7 @@ class HighsTaskExecutor {
     for (int i = 1, numThreads = static_cast<int>(workerDeques.size());
          i < numThreads; ++i) {
       workerThreads.emplace_back(
-          std::move(std::thread(&HighsTaskExecutor::run_worker, i, this)));
+          std::thread(&HighsTaskExecutor::run_worker, i, this));
```

Comment:
Moving a temporary prevents copy elision. This is from `Wpessimizing-move` from `clang`.